features/old/accounts/bulk_operations.feature

```diff
@@ -17,28 +17,23 @@ Feature: Bulk operations
     Given current domain is the admin domain of provider "foo.3scale.localhost"
     Given I am logged in as provider "foo.3scale.localhost"
 
-  # FIXME: THREESCALE-7195 this scenario is failing in CircleCI. We need to refactor it as an integration test.
-  @wip
   Scenario: Show and hide bulk operations controls
     When I navigate to the accounts page
     When I check select for "bob"
-    Then "Bulk operations" should be visible
+    And checks if bulk-operations the div is visible
       And I should see "Send email"
       And I should see "Change account plan"
       And I should see "Change state"
     When I uncheck select for "bob"
-    Then "Bulk operations" should not be visible
+    And checks if bulk-operations the div is hidden
 
-  # FIXME: THREESCALE-7195 this scenario is failing in CircleCI. We need to refactor it as an integration test.
-  @wip
   Scenario: Check all accounts with main checkbox
       And I am on the accounts admin page
 
     When I check select in table header
-
-    Then all selects should be checked
-      And "Bulk operations" should be visible
+    Then I check if all checkboxes are selected
```

Comment:
This step doesn't exist, so the old name `all selects should be checked` is available. Why did you rename it? I think the old one is better.

Reply:
ok